ports/stm32f4/boards/stm32f411ce_blackpill/README.txt

```diff
@@ -0,0 +1,3 @@
+Remember when creating the uf2 file that the family_id and base address are specified as follows:
```

Comment:
these should be part of the port readme, though, let just leave it here. I will move it later on if needed. At current stage, I haven't got any hard limit for bootloader size for stm32f4 port just yet.